templates/autocomplete.php

```diff
@@ -29,6 +29,9 @@
 				<span class="suggestion-post-content">{{{ data._snippetResult['content'].value }}}</span>
 			<# } #>
 		</div>
+		<?php
+		do_action( 'wp-search-with-algolia-autocomplete-after-content' );
```

Comment:
@tw2113 shouldn't this be `wp_search_with_algolia_autocomplete_after_content` with _ and not -?

Reply:
Is the prefix here correct?